python/ray/data/datasource/datasource.py

```diff
@@ -118,6 +118,16 @@ def on_write_failed(
         """
         pass
 
+    def get_name(self) -> str:
+        """Return a human-readable name for this datasource.
+        This will be used as the names of the read tasks.
+        """
+        name = type(self).__name__
+        datasource_suffix = "Datasource"
+        if name.endswith(datasource_suffix):
+            name = name[: -len(datasource_suffix)]
+        return f"Read{name}"
```

Comment:
Actually `Datasource` can do read and write. I think we should just return "<name>", instead of "Read<name>", o.w. it will become confusing.

Reply:
SGTM.